build/vagrant/buildspec-vagrant.yml

```
@@ -12,10 +12,10 @@ env:
 phases:
   install:
     commands:
-      - make -C build/vagrant install
+      - make -C -f Makefile.spel build/vagrant install
```

`-C` and `-f` actually conflict. No need to change these, since this is actually already telling make to use the vagrant buildspec.

```suggestion
      - make -C build/vagrant install
```